modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -836,49 +828,46 @@ GrainTracker::updateFieldInfo()
   std::map<unsigned int, Real> tmp_map;
   MeshBase & mesh = _mesh.getMesh();
 
-  for (std::map<unsigned int, MooseSharedPointer<FeatureData> >::iterator grain_it = _unique_grains.begin(); grain_it != _unique_grains.end(); ++grain_it)
+  for (const auto & grain_kv : _unique_grains)
   {
-    unsigned int curr_var = grain_it->second->_var_idx;
+    unsigned int curr_var = grain_kv.second->_var_idx;
```

Comment:
Another good candidate for catching by `auto`... in case the type of `_var_index` changes in the future (like to `short` or `long`)